src/LiveComponent/doc/index.rst

```diff
@@ -1824,14 +1824,24 @@ Now, create a Twig component to render the form::
         }
     }
 
-There is no need for a custom template just render the form as usual:
+Create a custom template for the component :
 
 .. code-block:: html+twig
 
+    {# templates/components/blog-post-collection-type.html.twig #}
```

Comment:
I like that you added the template path. Just make sure the filename matches whatever we end up calling the component class - e.g. `BlogPostForm`.